config/kibana.yml

```diff
@@ -133,6 +133,10 @@
 #  - name: metrics.ops
 #    level: debug
 
+# Enables debug logging on the browser (dev console)
+#logging.browser.root:
+#  level: debug
```

Comment:
Not sure it was really needed given right now it's mostly for internal/development usage, but I still added a snippet in the config file.